website/config.toml

```diff
@@ -119,8 +119,12 @@ disableKinds = ["taxonomy", "taxonomyTerm"]
   # These entries appear in the drop-down menu at the top of the website.
   [[params.versions]]
     version = "latest"
-    githubbranch = "release-v1.6.1-aws-b1.0.0"
+    githubbranch = "release-v1.7.0-aws-b1.0.0"
     url = "https://awslabs.github.io/kubeflow-manifests/docs"
+  [[params.versions]]
+    version = "v1.7"
+    githubbranch = "release-v1.7.0-aws-b1.0.0"
+    url = "https://awslabs.github.io/kubeflow-manifests/release-v1.7.0-aws-b1.0.0/docs"
   [[params.versions]]
     version = "v1.6"
     githubbranch = "release-v1.6.1-aws-b1.0.0"
```

Comment:
interesting, this should have been `release-v1.6.1-aws-b1.0.2` same for url, can you please update?

Reply:
addressed.